templates/terraform/resource.erb

```diff
@@ -143,7 +155,12 @@ func resource<%= resource_name -%>Read(d *schema.ResourceData, meta interface{})
 		return handleNotFoundError(err, d, fmt.Sprintf("<%= resource_name -%> %q", d.Id()))
 	}
 <% properties.each do |prop| -%>
-	if err := d.Set("<%= Google::StringUtils.underscore(prop.name) -%>", flatten<%= resource_name -%><%= titlelize_property(prop) -%>(res["<%= prop.api_name -%>"])); err != nil {
+  <% resource_plus_prop = resource_name + titlelize_property(prop) -%>
+  <% if object.custom_code.decoder.key? prop.name -%>
```

Comment:
Same, move this to the properties themselves.

Reply:
This one is getting global-ized but the others, yeah.